synapse/lib/stormtypes.py

```diff
@@ -1534,6 +1534,8 @@ async def _raise(self, name, mesg, **info):
         name = await tostr(name)
         mesg = await tostr(mesg)
         info = await toprim(info)
+        s_common.reqjsonsafe(info)
```

Comment:
this is a behavior change for raise since legit users raising a binary blob would now fail